Adafruit_AMG88xx.cpp

```diff
@@ -254,25 +246,18 @@ void Adafruit_AMG88xx::read(uint8_t reg, uint8_t *buf, uint8_t num) {
 }
 
 void Adafruit_AMG88xx::write(uint8_t reg, uint8_t *buf, uint8_t num) {
+  uint8_t prefix[1] = {reg};
 #ifdef I2C_DEBUG
```

Comment:
remvoe the i2cdebug stuff - we have i2c debugging enablable in busio (you can start to see all the things that busio does that this library did half-assed :)

Reply:
totes.